engine/gamesys/src/gamesys/scripts/script_resource.cpp

```diff
@@ -1095,6 +1095,8 @@ static int CreateTextureAsync(lua_State* L)
 
     dmGraphics::TextureCreationParams texture_create_params;
     texture_create_params.m_Type = create_params.m_Type;
+    texture_create_params.m_Width = create_params.m_Width;
+    texture_create_params.m_Height = create_params.m_Height;
```

Comment:
Crash fix: If the texture is created without width and height, the "original" width and height was never set, causing GIUs to crash when using it for some things like slice-9. It's not completely clear to me why we need these original width and heights, but it fixes the crash at least.